src/components/faqTemplates/ExpandableWithCategories.jsx

```diff
@@ -52,60 +15,55 @@ const TitleImg = styled.img`
 // FAQ Section with two columns with layout:
 // General    Logistics
 //            Teams & Projects
-const FaqSection = props => {
-  const [generalBin, setGeneralBin] = useState([])
-  const [logisticsBin, setLogisticsBin] = useState([])
-  const [teamsBin, setTeamsBin] = useState([])
+const ExpandableWithCategories = ({ faq, config }) => {
+  const [categorizedFaqMap, setFaqMap] = useState(new Map())
+
+  const categorizeFaq = faq => {
+    faq.forEach(({ category, question, answer }) => {
+      try {
```

Comment:
What kind of things are we trying to catch here?

Reply:
Oh, I didn't commit my latest change which removed this block. I'll also remove the one from useEffect as well